package/yast2-ruby-bindings.spec

```diff
@@ -39,8 +39,10 @@ Requires:       rubygem(%{rb_default_ruby_abi}:fast_gettext)
 %endif
 BuildRequires:  ruby-devel
 # new ErrorNamespace needed
-Requires:       yast2-core >= 3.2.1
-BuildRequires:  yast2-core-devel >= 3.2.1
+Requires:       yast2-core >= 3.2.3
+BuildRequires:  yast2-core-devel >= 3.2.3
+# replacement of y2base with its ruby implementation
+Conflicts:      yast2-core <= 3.2.2
```

Comment:
I completely missed the fact that we already require a recent version of the package. That implies conflicting with lower versions, so Conflicts can be removed.